sources/api/schnauzer/Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1"
 snafu = "0.6"
 tokio = { version = "0.2", default-features = false, features = ["macros", "rt-threaded"] }
+url = "2.2.0"
```

Comment:
nit: looks like 2.2 *just* came out.  Can we stick with 2.1 to match other packages, until we do our next bump?